src/User.js

```diff
@@ -9,60 +9,29 @@ class User {
     this.mySpending = this.getMySpending(tapeChart);
   }
 
-getMyBookings(tapeChart) {
- this.myBookings = tapeChart.bookings.filter(booking =>
-   booking.userID === this.id)
-   return this.myBookings;
-}
-
-getMySpending(tapeChart) {
-  this.mySpending = this.myBookings.reduce((acc, booking) => {
-    tapeChart.rooms.forEach(room => {
-      if (booking.roomNumber === room.number) {
-        acc += room.costPerNight;
-      }
-    })
-    return acc
-  }, 0)
-  return this.mySpending;
-}
-
-// index.js when selects date, fire tapeChart.getAvailableRoomsByDate() and display
-// if this function is invoked on user clicking given room, then i wouldn't
-// need the conditional to check if that room is available
-// based on select room, room # and date will autopopulate
-makeBooking (date, roomNumber) {
-  let booking = new Booking(this.id, date, roomNumber);
-  return booking;
-}
-
-// return bookingobj
-// instantiate new booking
-// let newBooking = new Booking(this.id, date, roomNumber)
-// return newBooking
-// invoke postBooking (object to post)
-// does the getMySpending need to calc past vs future?
-
-// PSUEDOCODING
-// select date - Dom
-// see what rooms available for date - tapeChart
-// select room from the available rooms
-// create Booking obj {roomNumber: 1 } - user
-// fetch POST that bookingobj
-// update tapechart stats
-// update user stats
-
-
-
-// Could have a getMyBookings method that takes TapeChart
-// instantiation as an arg then does the calculations here
-// within User which might be a little more SRP
+  getMyBookings(tapeChart) {
+   this.myBookings = tapeChart.bookings.filter(booking =>
+     booking.userID === this.id)
+     return this.myBookings;
+  }
 
-// could have a method for update bookings for after posting
-// date.  This could be in tapeChart and update the tapeCharts
-// this.bookings property then reinvoke the user's getMyBookings
-// method to update the users myBookings property.
+  getMySpending(tapeChart) {
+    this.mySpending = Number(this.myBookings.reduce((acc, booking) => {
+      tapeChart.rooms.forEach(room => {
+        if (booking.roomNumber === room.number) {
+          acc += room.costPerNight;
+        }
+      })
+      return acc
+    }, 0).toFixed());
```

Comment:
I think you will need to add a number in here like toFixed(2)

Reply:
Its wrapped in `Number` on line 19.  Is that what you mean?